lib/index.js

```diff
@@ -39,6 +46,28 @@ export default function runContentfulImport (usageParams) {
     return Promise.reject(new Error('Either the `contentFile` or `content` option are required.'))
   }
 
+  const infoTable = new Table()
+
+  infoTable.push([{colSpan: 2, content: 'The following amount of entities will be imported:'}])
```

Comment:
I would show the summary after the import is done 
and instead of `The following amount of entities will be imported:` -> `the following entities were imported`